### PR TITLE
added fetch call logs

### PR DESCRIPTION
**Pull request by chandumlg**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: util/customTransformer.js [hunk 1]
```diff
@@ -2,8 +2,12 @@ const ivm = require("isolated-vm");
 const fetch = require("node-fetch");
 const { getTransformationCode } = require("./customTransforrmationsStore");
 const { userTransformHandlerV1 } = require("./customTransformer-v1");
+const logger = require("../logger");
```
Comment (arnab-p) [inline util/customTransformer.js:5]: Redundant `require`

File: util/customTransformer.js [hunk 2]
```diff
@@ -18,17 +22,25 @@ async function runUserTransform(events, code, eventsMetadata) {
   await jail.set(
     "_fetch",
     new ivm.Reference(async (resolve, ...args) => {
+      const currTime = new Date().getTime()
```
Comment (arnab-p) [inline util/customTransformer.js:25]:
```suggestion
      const currTime = new Date().getTime();
```

File: util/customTransformer.js [hunk 3]
```diff
@@ -18,17 +22,25 @@ async function runUserTransform(events, code, eventsMetadata) {
...
       try {
         const res = await fetch(...args);
         const data = await res.json();
         resolve.applyIgnored(undefined, [
           new ivm.ExternalCopy(data).copyInto()
         ]);
       } catch (error) {
+        if (logEnabled) {
+          console.log("Got error: " + JSON.stringify(error) + " Events Metadata: " + eventsMetadata)
```
Comment (arnab-p) [inline util/customTransformer.js:34]:
```suggestion
          console.log("Got error: " + JSON.stringify(error) + " Events Metadata: " + eventsMetadata);
```

File: util/customTransformer.js [hunk 4]
```diff
@@ -18,17 +22,25 @@ async function runUserTransform(events, code, eventsMetadata) {
...
+        }
         resolve.applyIgnored(undefined, [
           new ivm.ExternalCopy("ERROR").copyInto()
         ]);
       }
+      if (logEnabled) {
+        diff = new Date().getTime() - currTime
```
Comment (arnab-p) [inline util/customTransformer.js:41]:
```suggestion
        const diff = new Date().getTime() - currTime;
```

File: util/customTransformer.js [hunk 5]
```diff
@@ -18,17 +22,25 @@ async function runUserTransform(events, code, eventsMetadata) {
...
+        console.log("Took: " + diff + " Events Metadata: " + eventsMetadata)
```
Comment (arnab-p) [inline util/customTransformer.js:42]:
```suggestion
        console.log("Took: " + diff + " Events Metadata: " + eventsMetadata);
```

File: util/ivmFactory.js [hunk 6]
```diff
@@ -128,17 +130,25 @@ async function createIvm(code, libraryVersionIds) {
   await jail.set(
     "_fetch",
     new ivm.Reference(async (resolve, ...args) => {
+      const currTime = new Date().getTime()
```
Comment (arnab-p) [inline util/ivmFactory.js:133]:
```suggestion
      const currTime = new Date().getTime();
```

File: util/ivmFactory.js [hunk 7]
```diff
@@ -128,17 +130,25 @@ async function createIvm(code, libraryVersionIds) {
...
       try {
         const res = await fetch(...args);
         const data = await res.json();
         resolve.applyIgnored(undefined, [
           new ivm.ExternalCopy(data).copyInto()
         ]);
       } catch (error) {
+        if (logEnabled) {
+          console.log("Got error: " + error)
```
Comment (arnab-p) [inline util/ivmFactory.js:142]:
```suggestion
          console.log("Got error: " + error);
```

File: util/ivmFactory.js [hunk 8]
```diff
@@ -128,17 +130,25 @@ async function createIvm(code, libraryVersionIds) {
...
+        }
         resolve.applyIgnored(undefined, [
           new ivm.ExternalCopy("ERROR").copyInto()
         ]);
       }
+      if (logEnabled) {
+        diff = new Date().getTime() - currTime
```
Comment (arnab-p) [inline util/ivmFactory.js:149]:
```suggestion
        const diff = new Date().getTime() - currTime
```

File: util/ivmFactory.js [hunk 9]
```diff
@@ -128,17 +130,25 @@ async function createIvm(code, libraryVersionIds) {
...
+        console.log("Time took in ms: " + diff)
```
Comment (arnab-p) [inline util/ivmFactory.js:150]:
```suggestion
        console.log("Time took in ms: " + diff);
```

**Review by arnab-p — APPROVED** [review]

**Comment by saikumarrs** [comment]
Closing the PR as it is stale.

**Pull request closed without merging**